pkg/trace/config/subscriber.go

```diff
@@ -18,11 +18,12 @@ type Subscriber struct {
 	configs        pbgo.ConfigResponse
 	once           sync.Once
 	stopSubscriber context.CancelFunc
+	tracerInfos    chan *pbgo.TracerInfo
```

Comment:
Can we please rename to `infoChan`? Or, whatever you like, as long as we add the suffix `Chan`.